prime-router/metadata/valuesets/common.valuesets

```diff
@@ -90,20 +90,28 @@
   values:
     - display: American Indian or Alaska Native
       code: 1002-5
+      version: 2.5.1
```

Comment:
Seems a shame we can't give the version for the whole valueSet.   its a big gloppy this way.

Reply:
Right. The version can change within the same value set.